sql/core/src/main/scala/org/apache/spark/sql/scripting/SqlScriptingExecutionNode.scala

```diff
@@ -75,14 +76,17 @@ trait NonLeafStatementExec extends CompoundStatementExec {
       //  of boolean type with value True.
       val df = Dataset.ofRows(session, statement.parsedPlan)
       df.schema.fields match {
-        case Array(field) if field.dataType == BooleanType =>
+        case Array(StructField(_, BooleanType, _, _)) =>
```

Comment:
why do we change this?

Reply:
Let's also fix `@return Whether the condition evaluates to True.` doc

Reply:
> why do we change this?

Hmm..., actually, it has nothing to do with this PR.
IMO it just looks more concise, 
Okay, let me restore it.

Reply:
> Let's also fix `@return Whether the condition evaluates to True.` doc

Updated.